assets/src/scripts/monitoring-location/components/map/legend.js

```diff
@@ -75,42 +87,6 @@ export const createFIMLegend = function(legendControl, isFIMAvailable) {
                         });
             });
     } else {
-        select(legendControl.getContainer()).select('#fim-legend-list').remove();
-    }
-};
-
-/**
- * Creates the NLDI legend if NLDI data is available, otherwise removes the NLDI legend if it exists.
- * @param {L.Control} legendControl - Leaflet legend control
- * @param {Boolean} is NLDI available
- */
-export const createNldiLegend = function(legendControl, isNldiAvailable) {
```

Comment:
I moved this to nldi-mapping as it needed to share styling constants with the code in that module.